README.md

```diff
@@ -1,93 +1,284 @@
 
 Federated projects cannot expect that the community will maintain or support the project as long as the project remains federated. If/when a federated project transitions to the community maintenance model, this project will be maintained and supported by the community.
 
 Federated projects are under no obligation to have a transition plan or any intention of becoming a community maintained project. However, this is strongly encouraged and community maintained projects are seen as the ideal. Federated projects are seen as the reality.
 
-## Incubation
+## Trademarks
+
+[Community Projects](#community-projects) need to license/assign the project's trademarks when becoming a conda Organization Community Project. This is an act of good faith and communicates to the community that the project are committed to our overall efforts.
 
-All projects start their journeys in an experimental or unstable state. To this end, we delineate stable projects as living in the `conda` GitHub organization. Therefore, projects new to the conda organization and governance must begin their inclusion here in the `conda-incubator` organization. The barrier to entry for a project to be added to the incubator should be very low. A vote from the Steering Council is required to move a project from incubation to the main GitHub organization.
+1. Trademarks are irrevocably *licensed or transferred* to the conda Organization's fiscal sponsor when a project is added to the conda Organization as a Community Project.
+2. The conda Organization, in conjunction with our fiscal sponsor, then have the right to sublicense the trademarks to other projects, subject to specific limitations: The conda Organization may not degrade the distinctiveness of *licensed* trademarks, or use *licensed* trademarks to disparage or misrepresent the assigner. *Transferred* trademarks are completely controlled by the conda Organization and its fiscal sponsor.
+3. The [Steering Council](#steering-council) then makes trademark decisions, subject to agreed upon restrictions, and coordinates efforts with our fiscal sponsor.
 
-Any single member of the Steering Council may add a project to the incubator, given the knowledge and consent of the author(s) (see [below](#voting)). Members of the Steering Council that are sponsoring the incubation are responsible for ensuring that basic legal requirements have been met that allow for the incubation. For example, trademark usage and license terms must been checked prior to incubation.
+We have guidelines for making decisions about conda Organization trademarks:
 
+1. The project should be open source with a permissive license
```

Comment:
We could include text [based on the description from Wikipedia](https://en.wikipedia.org/wiki/Permissive_software_license):

`1. The project should be open source with a permissive license.  A permissive license is a free-software license which instead of copyleft protections, carries only minimal restrictions on how the software can be used, modified, and redistributed.`

Would that be sufficient?

Reply:
@tnabtaf Does this exclude GPL or AGPL? I forget the definitions here. Excluding AGPL is fine by me but GPL should be allowed IMHO.

Reply:
@CJ-Wright: The PR has been updated to explicitly include the definition of permissive that is used by OSI.

@beckermr: Several of us discussed GPL/copyleft at SciPy and none of us were eager to embrace the potential legal complications that come with copyleft.